docs/documentation/make-first-prototype/link-index-page-start-page.md

```diff
@@ -5,7 +5,3 @@ You can route users from your service's index page to your start page. The index
 1. Open the `index.html` file in your `app/views` folder.
 2. Add an `<a>` tag that links to `/start`.
 
-You can now find out how to:
-
-- [redirect users based on their answers](https://govuk-prototype-kit.herokuapp.com/docs/creating-routes)
-- [publish your prototype online](https://govuk-prototype-kit.herokuapp.com/docs/publishing-on-heroku) to share it with your team or do user research
```

Comment:
I'm not sure if the changes made in this file are related to this PR.

Reply:
@hannalaakso I'll remove those changes...